api/src/lib.rs

```diff
@@ -1844,10 +1844,14 @@ impl Endpoints {
     }
 }
 
+const DEFAULT_HTTP_TIMEOUT: Lazy<Option<Duration>> = Lazy::new(|| Some(Duration::from_secs(120)));
```

Comment:
How about just making this
```rust
const DEFAULT_HTTP_TIMEOUT_SECONDS: u32 = 120
```
and then constructing the `Duration` in `.timeout()`?